atmo/clusters/models.py

```diff
@@ -116,6 +116,31 @@ def __str__(self):
     def __repr__(self):
         return "<Cluster {} of size {}>".format(self.identifier, self.size)
 
+    @property
```

Comment:
Yeah, this is about readability, I always considered properties to belong right after the magic methods, will fix the commit message accordingly